### PR TITLE
Login messages config

### PR DESCRIPTION
**Pull request by fastiuk**
This PR depends on https://github.com/fastiuk/sonic-buildimage/pull/4

#### Why I did it
* Add an ability to configure login messages: /etc/motd, /etc/issue*
* Be able to use table defines from swsscommon

#### How to verify it
Build SONiC
or
When SONiC was already built:
```
rm -f target/python-wheels/bullseye/sonic_yang_models-1.0-py3-none-any.whl
BLDENV=bullseye make -f Makefile.work target/python-wheels/bullseye/sonic_yang_models-1.0-py3-none-any.whl
rm -f target/python-wheels/bullseye/sonic_host_services-1.0-py3-none-any.whl
BLDENV=bullseye make -f Makefile.work target/python-wheels/bullseye/sonic_host_services-1.0-py3-none-any.whl
```
Login into the switch.
Set messages:
```
redis-cli -n 4 HSET "LOGIN_MESSAGE|pre_login" "message" "Test pre login"
redis-cli -n 4 HSET "LOGIN_MESSAGE|post_login" "message" "Test post login"
```
Logout/login and see the results:
Screenshot 2022-07-02 at 22 58 11
Clear messages:
```
redis-cli -n 4 HSET "LOGIN_MESSAGE|pre_login" "message" ""
redis-cli -n 4 HSET "LOGIN_MESSAGE|post_login" "message" ""
```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

#### Description for the changelog
* Add an ability to configure pre-login and post-login messages: /etc/motd, /etc/issue*

#### Link to config_db schema for YANG module changes
[sonic-login_message.yang](https://github.com/fastiuk/sonic-buildimage/tree/login-message-config/src/sonic-yang-models/yang-models/sonic-login_message.yang)

#### Superproject PR:
| PR title | State |
| ----------------- | ----------------- |
| [Login messages config](https://github.com/fastiuk/sonic-buildimage/pull/4) | GitHub issue/pull request detail |

### CODE REVIEW

**Comment by fastiuk** [comment]
@stepanblyschak please review

**Comment by fastiuk** [comment]
Hostcfgd moved here: https://github.com/fastiuk/sonic-host-services/pull/1

**Comment by stepanblyschak** [comment]
@fastiuk Please update PR to include only your changes

**Comment by fastiuk** [comment]
> Please update PR to include only your changes

Done

File: common/schema.h [hunk 1]
```diff
@@ -469,6 +472,11 @@ namespace swss {
 
 #define STATE_FLOW_COUNTER_CAPABILITY_TABLE_NAME    "FLOW_COUNTER_CAPABILITY_TABLE"
 
+#define STATE_EEPROM_INFO_TABLE_NAME        "EEPROM_INFO"
+#define STATE_FAN_INFO_TABLE_NAME           "FAN_INFO"
+#define STATE_TEMPERATURE_INFO_TABLE_NAME   "TEMPERATURE_INFO"
+#define STATE_PSU_INFO_TABLE_NAME           "PSU_INFO"
```
Comment (stepanblyschak) [inline common/schema.h:478]: These lines are not related to log message config

File: common/schema.h [hunk 2]
```diff
@@ -263,6 +263,7 @@ namespace swss {
 #define CFG_LAG_INTF_TABLE_NAME           "PORTCHANNEL_INTERFACE"
 #define CFG_VLAN_INTF_TABLE_NAME          "VLAN_INTERFACE"
 #define CFG_VLAN_SUB_INTF_TABLE_NAME      "VLAN_SUB_INTERFACE"
+#define CFG_MGMT_PORT_TABLE_NAME          "MGMT_PORT"
```
Comment (stepanblyschak) [inline common/schema.h:266]: This line is not related to log message config

Comment (fastiuk) [reply common/schema.h:266]: Yes, it doesn't.
That is why I put it in a separate commit.

File: common/schema.h (hunk 1 above)
Comment (fastiuk) [reply common/schema.h:478]: Yes, it doesn't.
That is why I put it in a separate commit.

File: common/schema.h (hunk 2 above)
Comment (stepanblyschak) [reply common/schema.h:266]: The PR title is "Login messages config". SONiC community requires to divide different features contribution into different PRs.
